examples/pytorch/gat/train.py

```diff
@@ -107,7 +107,8 @@ def main(args):
     for epoch in range(args.epochs):
         model.train()
         if epoch >= 3:
-            torch.cuda.synchronize()
```

Comment:
You should be able to replace `if cuda is True:` with `if cuda:`

Reply:
I changed it, I overwritten the original modification with git push -f, I don't know if this has any effect.